app/models/manageiq/providers/openstack/infra_manager.rb

```diff
@@ -64,6 +64,12 @@ def self.description
     @description ||= "OpenStack Platform Director".freeze
   end
 
+  def self.default_blacklisted_event_names
```

Comment:
@blomquisg It used to be in ```config/event_handling.tmpl.yml```. But this [PR](https://github.com/ManageIQ/manageiq/pull/3636) has moved them into DB.
Each provider model needs to maintain a list of the filtered events in ```default_blacklisted_event_names```.